conferences/Classes/PresentationLayer/UserStories/ReportsUserStory/Modules/ReportListModule/View/DataDisplayManager/ReportListDataDisplayManager.m

```diff
@@ -66,8 +66,11 @@ - (CGFloat)tableView:(UITableView *)tableView heightForRowAtIndexPath:(NSIndexPa
 }
 
 - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath *)indexPath {
-    EventPlainObject *event = [self.events objectAtIndex:indexPath.row];
-    [self.delegate didTapCellWithEvent:event];
+    // ignore header & footer
+    if (indexPath.row > 0 && indexPath.row <= self.events.count) {
```

Comment:
Instead of using `UITableViewDelegate` try NITableViewActions - it's the preferred way and you won't have to implement this nasty if-else.